docs/tutorials/mab.ipynb

```diff
@@ -20,7 +20,7 @@
     "from rich import print\n",
     "\n",
     "from pybandits.model import Beta\n",
-    "from pybandits.smab import SmabBernoulli, create_smab_bernoulli_cold_start"
+    "from pybandits.smab import SmabBernoulli"
    ]
   },
   {
```

Comment:
in cell [3] you can delete the line `n_objectives = 2`. It is not used below (I guess it's an old copy-past issue)